Foodhut/src/sections/introduction.html

```diff
@@ -19,12 +19,10 @@ <h1 class="header-introduction">
       Best cooks and best delivery guys all at your service. Hot tasty food will reach you in 60 minutes.
     </p>
     <form class="search-box-container flex between">
-      <div class="search-black-icon">
-        <img src="./assets/images/search-icon-black.svg" alt="search black icon"/>
-      </div>
+      <div class="search-black-icon"></div>
       <input class="search-container" type="text" placeholder="Search food" />
       <button class="search-white-icon" type="button">
-        <img src="./assets/images/search-icon-white.svg" alt="search white icon"/>
+        <div class="search-icon"><img src="./assets/images/search-icon-white.svg" alt="search white icon"/></div>
```

Comment:
Naming cause me confuse `search-white-icon` and `search-icon` what is the different?

Reply:
i fixed it

Reply:
@CBinh710 A suggestion for you, on element:
<button class=search-btn><img class=search-icon></button>
Make it simple

Reply:
i fixed it, thanks